### PR TITLE
Documentation for email@2.2

### PR DESCRIPTION
**Pull request by StorytellerCZ**

### CODE REVIEW

File: source/api/email.md [hunk 1]
```diff
@@ -69,3 +93,30 @@ if you want to intercept emails sent by core packages like accounts-password
...
+function with addition of `settings` key which will pass in package settings
+set in your app settings (if any). It is up to you what you do in that function
+as it will override the original sending function.
+
+Example:
+```javascript
+import { Email } from 'meteor/email'
+
+Email.customTransport = (options) => {
+  // `options.settings` are settings from `Meteor.settings.packages.email`
```
Comment (filipenevola) [inline source/api/email.md:114]: I believe this comment is wrong now, it should be packageSettings right?

File: source/api/email.md [hunk 2]
```diff
@@ -69,3 +93,30 @@ if you want to intercept emails sent by core packages like accounts-password
...
+
+{% apibox "Email.customTransport" %}
+
+> `Email.customTransport` is only available since Email v2.2
+
+There are scenarios when you have your own transport set up, be it an SDK 
+for your mailing service or something else. This is where `customTransport` 
+comes in. If you set this function all sending events will be passed to it
+(after `hookSend` is run) with an object of the options passed into `send` 
+function with addition of `settings` key which will pass in package settings
```
Comment (filipenevola) [inline source/api/email.md:105]: `settings` here I believe should be `packageSettings` as well, right?

**Review by filipenevola — CHANGES_REQUESTED** [review]

File: source/api/email.md (hunk 1 above)
Comment (StorytellerCZ) [reply source/api/email.md:114]: Fixed

File: source/api/email.md (hunk 2 above)
Comment (StorytellerCZ) [reply source/api/email.md:105]: fixed

**Review by filipenevola — APPROVED** [review]

**Pull request merged**